modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/formulaFieldUtil.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export type FormulaOutput = {
+	description: string;
+	label: string;
+	value: string;
+};
+
+export const FORMULA_OUTPUT_OPTIONS: FormulaOutput[] = [
+	{
+		description: Liferay.Language.get('select-between-two-possible-values'),
+		label: Liferay.Language.get('boolean'),
+		value: 'boolean',
+	},
+	{
+		description: 'for-date-values',
```

Comment:
Hi @danvb112. I think here should be:

```suggestion
		description: Liferay.Language.get('for-date-values'),
```

Reply:
Thank you for seeing this @alinedoleron! Done! :smile: